docs/topics/secured-mission-nodejs-adapter.adoc

```diff
@@ -0,0 +1,50 @@
+= SSO Adapter Configuration
+
+The SSO adapter is the _client side_, or client to the SSO server, component that enforces security on the web resources. In this specific case, it is the Greeting service.
+
+
+.Enacting Security Example {NodeJS} Code
+[source,javascript,options="nowrap",subs="attributes+"]
+----
+const express = require(‘express’);
+const Keycloak = require(‘keycloak-connect’); <1>
+const kc = new Keycloak({});  <2>
+
+const app = express();
+
+app.use(kc.middleware());  <3>
+
+app.use(‘/api/greeting’, kc.protect(), callback);  <4>
+----
+
+<1> `npm` module `keycloak-connect` must be installed and `required`.
+<2> Instantiate a new `Keycloak` object and pass in an empty configuration object.
+<3> Tells `express` to use Keycloak as middleware.
+<4> Enforces that a user must be authenticated before accessing a resource.
+
+
+.Enacting Security in Keycloak Adapter using `keycloak.json`
+[source,json,options="nowrap",subs="attributes+"]
+----
+{
+  "realm": "master", <1>
```

Comment:
Same comment here about line numbers. Although since JSON doesn't allow comments, I'm not sure how you would do this much differently.